<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinosPage</class>
 <widget class="QWidget" name="ArduinosPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="verticalFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Button" name="sensors_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 15px;
	padding: 5px;
	background-color: rgb(64, 68, 237);
	color: white;
	border: 2px solid rgba(0, 0, 0, 10); /* Borde tenue */
}

QPushButton:hover {
  background-color: rgb(90, 90, 255);    /* Fondo al pasar el mouse */
  color: white;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(51, 55, 188)     /* Fondo al presionar */
  color: white;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
        </property>
        <property name="text">
         <string>Sensores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Button" name="machines_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 15px;
	padding: 5px;
	background-color: rgb(64, 68, 237);
	color: white;
	border: 2px solid rgba(0, 0, 0, 10); /* Borde tenue */
}

QPushButton:hover {
  background-color: rgb(90, 90, 255);    /* Fondo al pasar el mouse */
  color: white;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(51, 55, 188)     /* Fondo al presionar */
  color: white;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
        </property>
        <property name="text">
         <string>Equipos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Button" name="registers_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 15px;
	padding: 5px;
	background-color: rgb(64, 68, 237);
	color: white;
	border: 2px solid rgba(0, 0, 0, 10); /* Borde tenue */
}

QPushButton:hover {
  background-color: rgb(90, 90, 255);    /* Fondo al pasar el mouse */
  color: white;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(51, 55, 188)     /* Fondo al presionar */
  color: white;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
        </property>
        <property name="text">
         <string>Registros</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>5</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="subpages_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	border: 2 px solid black;
}
</string>
     </property>
     <layout class="QGridLayout" name="subpages_layout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>view/elements/button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
